reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
+---------------
 
 For compiling under Windows, Linux or OSX, the following is required:
 
--  Python 2.7+ (3.0 is untested as of now).
--  SCons build system.
+-  Python 2.7+ [3.0 is untested as of now]
+-  pywin32 [Optional, Windows only. Applied when -j(number of cores) specified]
+-  SCons build system
 -  Android SDK version 19 [Note: Please install all Tools and Extras of sdk manager]
 -  Android build tools version 19.1
--  Android NDK
--  Gradle
--  OpenJDK 6 or later (or Oracle JDK 6 or later)
+-  Android NDK r10e or later
+-  Gradle [will be downloaded and installed when compiling for android if it doesn't exist]
+-  JDK 6+ (Oracle's releases) or OpenJDK 6+
```

Comment:
We are all know that Godot Engine is Open Source. Fortunately, I can confirm of being an Open Source fans. Dangerously, Godot is not just for Open Source people, instead for both Open Source and/or Propietary people, coming from GameMaker Studio, Unity, GameSalad, Construct, and so on.

OpenJDK 6+ or JDK 6+ == JDK 6+ or OpenJDK 6+

I'm sorry, but I believe it has nothing to do with which one should be implemented first (A first, B later), as it's not something like scons first and then gradle. It's about choice, which saying we can use A or B (B or C).

Now, why I re-arrange the position from OpenJDK or JDK to JDK or OpenJDK? It's simply because : 

- I guess JDK (mostly windows, mac users?) way more popular than OpenJDK. So, readers (especially new comers) could understand it quickly.
- Android SDK Docs does not even mention about OpenJDK (while now we are discussing about polishing android related docs, right?) The one mentioned by Google team for Windows, Mac, and Linux users is Oracle Java Development Kit (JDK) only, and none for OpenJDK (yeah, I can agree we should boom Google offices, for not "loving" OpenSource community.) 😄 

Again... It's all up to your style and understanding, because you're the boss after my prestigious men, Juan Linietsky and Ariel Manzur.

:-)